packages/amplify_datastore/example/lib/main.dart

```diff
@@ -85,7 +85,11 @@ class _MyAppState extends State<MyApp> {
   // Platform messages are asynchronous, so we initialize in an async method.
   Future<void> initPlatformState() async {
     try {
-      datastorePlugin = AmplifyDataStore(modelProvider: ModelProvider.instance);
+      datastorePlugin = AmplifyDataStore(
+        modelProvider: ModelProvider.instance,
+        errorHandler: ((error) =>
+            {print("Custom ErrorHandler result: " + error.toString())}),
```

Comment:
mega nit: "result" sounds like we are expecting some thing to be returned intentionally, could be "Custom ErrorHandler received error" :D

Reply:
nice good point!  updated